readthedocs/rtd_tests/tests/test_celery.py

```diff
@@ -123,3 +130,56 @@ def test_sync_repository(self):
                 args=(version.pk,),
             )
         self.assertTrue(result.successful())
+
+    @patch('readthedocs.projects.tasks.api_v2')
+    def test_check_duplicate_reserved_version_latest(self, api_v2):
```

Comment:
I think this one and the next one are similar enough to use `pytest.mark.parametrize` with `stable` and `latest` as arguments: https://docs.pytest.org/en/latest/parametrize.html

Also, instead of `stable` and `latest` use the `LATEST` and `STABLE` constants.

Reply:
We can't use the parametrize feature in class-based tests, sadly https://docs.pytest.org/en/latest/unittest.html#pytest-features-in-unittest-testcase-subclasses

Reply:
:(

What about creating a `_helper` method with the duplicated code and call it twice from the test?